solc/CommandLineInterface.cpp

```diff
@@ -869,7 +869,7 @@ General Information)").c_str(),
 			g_strEVMVersion.c_str(),
 			po::value<string>()->value_name("version"),
```

Comment:
```suggestion
			po::value<string>()->value_name("version")->default_value(EVMVersion::berlin().name()),
```
Then you can remove the `(default)` part from the description below because `boost::program_options` will automatically show it on the help screen.